examples/simple/fxos-icons/fxos-icons.css

```diff
@@ -0,0 +1,45 @@
+/* Generated by grunt-webfont */
```

Comment:
Is this file a dependency it should be sourced from `node_modules`. You may have to use a local web-server to run the demo to prevent cross-origin font issues.